client/src/app/site/site.component.html

```diff
@@ -29,11 +29,20 @@
                     <span> {{ getLangName(this.translate.currentLang) }} </span>
                 </a>
                 <div *ngIf="isLoggedIn">
-                    <a (click)="editProfile()" mat-list-item>
+                    <a
+                        routerLink="/users/{{ this.operator.user.id }}"
+                        (click)="mobileAutoCloseNav()"
+                        mat-list-item
+                    >
                         <mat-icon>person</mat-icon>
                         <span translate>Edit profile</span>
```

Comment:
@tsiegleauq Isn't it a `show/view profile`, because one  does not land there in edit mode. Maybe we should change this wording.